content/source/docs/enterprise/sentinel/mock.html.md

```diff
@@ -0,0 +1,157 @@
+* [Request a plan
+  export](/docs/enterprise/api/plan-exports.html#create-a-plan-export) using the
+  discovered plan ID. You will want to supply a Sentinel export type, such as
+  `sentinel-mock-bundle-v0`.
+* Monitor the export request by [viewing the plan
+  export](/docs/enterprise/api/plan-exports.html#show-a-plan-export). When the
+  status has reached `finished`, the export data is ready for download.
+* Finally, you can [download the export
+  data](/docs/enterprise/api/plan-exports.html#download-exported-plan-data). You
+  have up to an hour to fetch the data after an export request has been made,
+  after which the data is discarded and generation must be requested again.
+
+## Using Mock Data
+
+Mock data is generally supplied as a bundled tarball, containing 3 files:
+
+```
+mock-tfconfig.sentinel # tfconfig mock data
+mock-tfplan.sentinel   # tfplan mock data
+mock-tfstate.sentinel  # tfstate mock data
+```
+
+It's advised that these be stored in a subdirectory of the repository holding
+your policies, to ensure that they do not interfere with `sentinel test`. While
+the test data is Sentinel code, it's not a policy and will produce errors if
+evaluated like one.
+
+```
+.
+├── foo.sentinel
+├── sentinel.json
+├── test
+│   └── foo
+│       ├── fail.json
+│       └── pass.json
+└── testdata
+    ├── mock-tfconfig.sentinel
+    ├── mock-tfplan.sentinel
+    └── mock-tfstate.sentinel
+```
+
+Each configuration that needs access to the mock should reference the mock data
+files within the `mock` block in the specific Sentinel configuration file.
+
+For `sentinel apply`, this path is relative to the working directory. Assuming
+you always run this command from the repository root, the `sentinel.json`
+configuration file would look like:
+
+```
+{
+  "mock": {
+    "tfconfig": "testdata/mock-tfconfig.sentinel",
+    "tfplan": "testdata/mock-tfplan.sentinel",
+    "tfstate": "testdata/mock-tfstate.sentinel"
+  }
+}
+```
+
+For `sentinel test`, the paths are relative to the specific test configuration
+file. For example, the contents of `pass.json`, asserting that the result of the
+`main` rule was `true`, would read:
+
+```
+{
+  "mock": {
+    "tfconfig": "../../testdata/mock-tfconfig.sentinel",
+    "tfplan": "../../testdata/mock-tfplan.sentinel",
+    "tfstate": "../../testdata/mock-tfstate.sentinel"
+  },
+  "test": {
+    "main": true
+  }
+}
+```
+
+## Mock Data Availability
+
+The following factors can affect your ability to generate mock data:
+
+* You do not have [write
+  access](/docs/enterprise/users-teams-organizations/permissions.html#write) to
+  the workspace. This is to protect the possibly sensitive data which can be
+  procured via mock generation.
+* The run has not progressed past the planning stage, or did not create a plan
+  successfully.
+* The run has been in a terminal state, such as applied or discarded, longer
+  than seven days. At this point, the data necessary to generate the data is
```

Comment:
```suggestion
  than seven days. At this point, the data necessary to generate the plan export is
```

Reply:
I want to keep the context in the fields outside of the API focused on mock generation, so I will probably change "plan export" to "mock data" here again, to be specific.

Reply:
Changed it to "mocks" which reads better - seeing as you probably re-worded it to get rid of the stutter 😄